src/resources/lang/de/crud.php

```diff
@@ -125,16 +125,16 @@
     'browse_uploads' => 'Uploads durchsuchen',
     'clear' => 'Löschen',
     'page_link' => 'Link zur Seite',
-    'page_link_placeholder' => 'http://example.com/your-desired-page',
+    'page_link_placeholder' => 'http://example.com/ihre-gewuenschte-seite',
     'internal_link' => 'Interner Link',
-    'internal_link_placeholder' => 'Interner slug. Bsp: \'admin/page\' (ohne Anführungszeichen) für \':url\'',
+    'internal_link_placeholder' => 'Internes Kürzel. Bsp: \'admin/page\' (ohne Anführungszeichen) für \':url\'',
     'external_link' => 'Externer Link',
-    'choose_file' => 'Datei wählen',
+    'choose_file' => 'Datei auswählen',
 
     //Table field
     'table_cant_add' => 'Kann :entity nicht hinzufügen',
     'table_max_reached' => 'Maximale Anzahl von :max erreicht',
 
     // File manager
-    'file_manager' => 'Datei Manager',
+    'file_manager' => 'Datei-Manager',
```

Comment:
It's a compound word which should either be written with no separating character or separated with a hyphen. A space character is not correct in this case.